resources/layout/_menu.php

```diff
@@ -5,14 +5,30 @@
 use Yiisoft\Html\Html;
 
 /** @var \App\ApplicationParameters $applicationParameters */
+
+$currentUrl = $url->generate($urlMatcher->getCurrentRoute()->getName());
 ?>
-<div class="navbar">
-    <div class="navbar-brand">
-        <span class="navbar-item">
-            <img src="/images/yii-logo.jpg" alt="<?= Html::encode($applicationParameters->getName()) ?>">
-        </span>
-        <a href="/" class="navbar-item has-text-light">
-            <?= Html::encode($applicationParameters->getName()) ?>
-        </a>
+
+<nav id="w0-navbar" class="is-black navbar">
+    <div class="container">
+        <div class="navbar-brand">
+            <span class="navbar-item">
+                <img src="/images/yii-logo.jpg" alt=<?= Html::encode($applicationParameters->getName()) ?>>
+            </span>
+            <a class="navbar-item" href="/"><?= Html::encode($applicationParameters->getName()) ?></a>
+            <span class="navbar-burger burger" data-target="navbarMenu">
+                <span></span><span></span><span></span>
```

Comment:
Looks bad... is it necessary?

Reply:
I think it can be replaced with the `fas fa-bars` icon

Reply:
done